Jenkinsfile

```diff
@@ -25,9 +25,16 @@ def initializeEnvironment() {
     . ${JABBA_SHELL}
     jabba which 1.8''', returnStdout: true).trim()
 
+  env.TEST_JAVA_HOME = sh(label: 'Get TEST_JAVA_HOME',script: '''#!/bin/bash -le
+    . ${JABBA_SHELL}
+    jabba which ${JABBA_VERSION}''', returnStdout: true).trim()
+  env.TEST_JAVA_VERSION = sh(label: 'Get TEST_JAVA_HOME',script: '''#!/bin/bash -le
+    echo "${JABBA_VERSION##*.}"''', returnStdout: true).trim()
+
+
   sh label: 'Download Apache CassandraⓇ or DataStax Enterprise',script: '''#!/bin/bash -le
     . ${JABBA_SHELL}
-    jabba use ${JABBA_VERSION}
+    jabba use 1.8
```

Comment:
Shall we have a comment explaining the difference between JABBA_VERSION, TEST_JAVA_VERSION, etal?

Reply:
Done. I've also filed JAVA-3064 to tidy up how jabba is used in Jenkinsfile since after this change we no longer need the all the `JAVA_HOME` switching stuff in `buildDriver` and `executeTests`